growth/too/models.py

```diff
@@ -227,7 +231,8 @@ def create_all():
                                        reference_filter_mags=ref_filter_mags,
                                        ipix=ipix.tolist()))
 
-            if tele == "ZTF":
+            if False:
```

Comment:
This looks like some debugging code that was left in by mistake.

Reply:
oops

Reply:
Fixed.